### PR TITLE
Added Cloud answers

### PR DESCRIPTION
**Pull request by anwesh-b**
I have answered the unanswered questions of Cloud.
If there is anything to improve, do let me know

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -513,10 +517,11 @@ SAAS
 <summary>Explain each of the following Cloud Computing Deployments:
 
   * Public
-  * Hybrid
-  * Private</summary><br><b>
-
-  * Hybrid - combination of public and private clouds
+  * Private
+  * Hybrid</summary><br><b>
+  * Public - Cloud services having resources that is not shared among customers
```
Comment (bregman-arie) [inline README.md:522]: This is not the right answer

File: README.md [hunk 2]
```diff
@@ -513,10 +517,11 @@ SAAS
...
+  * Private - Cloud services sharing computing resources among multiple customers
```
Comment (bregman-arie) [inline README.md:523]: Also not the right definition of what is a Private cloud

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -513,10 +517,11 @@ SAAS
 <summary>Explain each of the following Cloud Computing Deployments:
 
   * Public
-  * Hybrid
```
Comment (bregman-arie) [inline README.md:516]: Hybrid was removed from the list

File: README.md [hunk 4]
```diff
@@ -499,6 +500,9 @@ SAAS
   * IAAS
   * PAAS
   * SAAS</summary><br><b>
```
Comment (bregman-arie) [inline README.md:502]: and don't need*

File: README.md [hunk 5]
```diff
@@ -499,6 +500,9 @@ SAAS
...
+  * IAAS - Users have control over complete Operating System and need not to worry about the physical resources, which is managed by Cloud Service Provider.
+  * PAAS - CLoud Service Provider takes care of Operating System, Middlewares and users only need to focus on our Data and Application.
```
Comment (bregman-arie) [inline README.md:504]: s/CLoud/Cloud/
s/focus on our Data and Application/focus on their application and its data/

File: README.md [hunk 6]
```diff
@@ -499,6 +500,9 @@ SAAS
...
+  * SAAS - Users only worry about the code and the Application, Data, OS and all stuffs are managed by Cloud Service Provider.
```
Comment (bregman-arie) [inline README.md:505]: Not accurate. Data, in any of the models, can reside locally or be managed by the cloud provider

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (anwesh-b) [reply README.md:516]: Just moved the order Public, Hybrid, Private > Public, Private, Hybrid

**Comment by anwesh-b** [comment]
I just deleted half of the readme

**Pull request closed without merging**